frontend/lib/src/components/input/input_field.dart

```diff
@@ -65,10 +69,12 @@ class InputField extends StatelessWidget {
           ),
           child: FormBuilderTextField(
             name: name,
+            initialValue: initialValue,
             validator: validator,
             controller: controller,
```

Comment:
@jstephend-sun sir good afternoon does controller is needed in this component sir? since we are using formbuilder? if not I think we can remove this one sa?

Reply:
@CaneteJoash where is controller even in this code change?

side comment, `does controller is` i think you should work on your grammar a bit,